app/src/main/java/org/groundplatform/android/ui/datacollection/tasks/polygon/DrawAreaTaskMapFragment.kt

```diff
@@ -50,4 +51,11 @@ class DrawAreaTaskMapFragment @Inject constructor() :
     taskViewModel.draftArea
       .map { feature: Feature? -> if (feature == null) setOf() else setOf(feature) }
       .asLiveData()
+
+  fun updateCenterMarker() {
+    super.updateCenterMarker(
+      isTooClose = taskViewModel.isTooClose,
+      isMarkedComplete = taskViewModel.isMarkedComplete(),
+    )
```

Comment:
Instead of calling super, this would be replaced with `setCenterMarkerVisibility`